### PR TITLE
BC-3135 Add file info for leaving team

### PR DESCRIPTION
**Pull request by blazejpass**
# Description
Add file listing while leaving team.

## Links to Tickets or other pull requests
server side update:
https://github.com/hpi-schul-cloud/schulcloud-server/pull/4078
ticket:
https://ticketsystem.dbildungscloud.de/browse/BC-3135

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: static/scripts/teams.js [hunk 1]
```diff
@@ -196,6 +197,25 @@ $(document).ready(() => {
 			payload: { userId },
 		});
 
+		const $deleteInfoAlert = $leaveTeamModal.find('.alert');
+		const $fileList = $leaveTeamModal.find('.file-list');
+
+		if (files && files.length > 0) {
```
Comment (VikDavydiuk) [inline static/scripts/teams.js:203]: U can simplify with `if (files?.length) {}`

File: static/scripts/teams.js [hunk 2]
```diff
@@ -196,6 +197,25 @@ $(document).ready(() => {
...
+			let fileListHtmlLi = '';
```
Comment (VikDavydiuk) [inline static/scripts/teams.js:204]: U can simplify with const fileListHtml = files.map(file => `<li>${file}</li>`).join(''); if it is array

File: static/scripts/teams.js (hunk 1 above)
Comment (blazejpass) [reply static/scripts/teams.js]: Changed according to suggestions

File: static/scripts/teams.js (hunk 2 above)
Comment (blazejpass) [reply static/scripts/teams.js]: Changed according to suggestion

**Review by VikDavydiuk — APPROVED** [review]

**Review by CeEv — APPROVED** [review]

**Pull request merged**